samples/snippets/csharp/api/system.reflection.metadata.ecma335/metadatabuilder/MetadataBuilderSnippets.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+

```

Comment:
I don't think we need the project settings in the examples, and never seen in any example

Reply:
Samples need to have a project file now according to guidelines: https://github.com/dotnet/samples#creating-new-samples For example, [MetadataReader](https://github.com/dotnet/dotnet-api-docs/tree/main/samples/snippets/csharp/api/system.reflection.metadata/metadatareader) sample has it.

Reply:
Thank you for the info, I was not aware of that, according to the guidelines apparently, you need to keep the `Program.cs` too, as the `Main entry point that calls your samples`.